docs/annotations.rst

```diff
@@ -1,4 +1,60 @@
 Showing annotations in WorldWide Telescope
 ==========================================
 
-.. here we need to demonstrate how to show e.g. circles or polygons
+Annotation objects are shapes that a user can manually add to the viewer in order to add another dimension of adaptability for a variety of uses in tours, presentations, and the like. Users can generate several annotations at once, choosing from circles, polygons, and lines.
```

Comment:
Just a minor comment - wrap the text so that it's no longer than 80 characters (most code editors provide a way to do this automatically, so no need to insert line breaks one by one!).

Reply:
Another minor style comment: I would maybe not say 'a user' - I would either say 'that can be manually added' or 'that you can manually add' (that is, don't refer to the user in the third person). This isn't a hard rule, just what I think reads well :) Same for the second sentence - you can either say 'Several annotations can be generated at once' or 'You can generate several annotations at once'. You can always mix the passive and 'you' so that you don't say 'you' too much.